### ISSUE TITLE
Error: For input string: "<my-saucelabs-key>@ondemand.saucelabs.com:80" (cinnamon) 

### ISSUE THREAD

**Issue by ssisson** [forcedotcom/cinnamon#40]
Hello, 

I followed the example in the setup doc and it seemed that I had no trouble getting things configured, however, when I go to execute a test using the unmodified demo code, the test failed with the following error:

For input string: "my-saucelabs-key@ondemand.saucelabs.com:80"
(cinnamon) 

The error message has my Saucelabs access key before the @ in the error message. It looks as though the webdriver might be trying to authenticate me and failing? I've verified that the key is correct and even tried generating a new one. 

Any help would be appreciated! Thanks!

**Comment by icemft76** [comment]
Noticed same issue - this has to be something basic but I can't figure it out. 

If the Saucelabs proxy or port was wrong would it reflect a more specific error message?

**Comment by nadeemg** [comment]
Are you guys getting this consistently? If yes, can you verify that you can reach ondemand.saucelabs.com in a browser?

**Comment by ssisson** [comment]
Yup, I've encountered it every time so far: 

![salesforce_com_-_developer_edition](https://cloud.githubusercontent.com/assets/11198368/6452435/b38eaa7c-c103-11e4-8313-9271343eac0a.gif)

And it looks like I can browse to ondemand.saucelabs.com 

![ondemand_saucelabs_com](https://cloud.githubusercontent.com/assets/11198368/6452448/c56f287a-c103-11e4-843c-d739dc2721e8.gif)

**Comment by icemft76** [comment]
same behavior as above ^^ happens on the TestNewAccount class

**Comment by nadeemg** [comment]
Thanks for the info. Bad news is that I'm still having trouble reproducing this. 

So one more request for you guys: when you run a test, could you also log into the saucelabs account you're using and see if a new session is actually being created? @ssisson @icemft76

**Comment by ssisson** [comment]
Hi,

It doesn't seem that a Saucelabs session is getting created. So for
whatever reason, we're not getting all the way there.

I'm pretty new to saucelabs-- do I need to configure a sauceconnect tunnel
or something like that (upgrade our license, etc) to get this to work?

Thank you!

**Comment by nadeemg** [comment]
@ssisson you shouldn't have to configure a sauceconnect tunnel. On my DE org, with a clean install, the sample is running fine. So, I'm wondering what the difference is between your org and mine. 

Did you have any trouble filling out the 'Settings' page? If you could remember your steps there that might help. 

Also, could you share your orgId? That would let me do some debugging on the Salesforce side.

**Comment by ssisson** [comment]
@nadeemg filling out the Setting page went great, although I recall now that there was an "endpoint" error that occurred in the Webdriver Service Provider Settings when I first installed. I had to go and add a new Remote Site setting to replace the one that was there and was failing. I made the original inactive and ran it with my replacement. The error went away and it seemed that I was able to reload the Sauce browser configs, so I assumed all was well. 

Just now, I attempted to disable my version of the Remote Site setting and return it to the original packaged version, but unfortunately the error remains.  

I emailed you my Salesforce Org ID. Let me know if there's any additional information I can provide. Thanks very much for your help!

**Comment by ssisson** [comment]
@nadeemg, I figured this out. It was as simple as you'd expect-- a typo in my Saucelabs username. 
I didn't catch it earlier because I was using what I thought was a "global save" at the top of the Settings page, not the nested one that went along with the Webdriver portion, so early on, when I noticed the typo and fixed it, I thought it had been saved but it hadn't.   

It's running great now!

Thanks for your help! Much appreciated.
